lua/quicker/treesitter.lua

```diff
@@ -0,0 +1,91 @@
+---@alias quicker.LangRegions table<string, number[][][]>
+
+local M = {}
+
+M.cache = {} ---@type table<number, table<string,{parser: vim.treesitter.LanguageTree, highlighter:vim.treesitter.highlighter, enabled:boolean}>>
```

Comment:
Anything that doesn't need to be accessible from outside the module shouldn't live on `M.`